CHANGELOG

```diff
@@ -1,3 +1,25 @@
+0.3.5
```

Comment:
multiple new versions in a single PR?

Reply:
Now at 0.3.7 ... :-) 
Feel free to set the version you feel okay.